parser_with_temp.py

```diff
@@ -75,35 +75,37 @@ def get_top_number_of_general_page(self):
             re.search('\d+\)', summary_pages).group(0)[:-1])
 
     def get_id_list(self):
-        """Iterates a list up to last_page_number, calling pages with general
-        tables and grabs all order id's into a list with all orders id's
+        """Iterates a range up to last_page_number, calling pages with general
+        tables and grabs all order id's into a list.
         :return a list with all orders numbers.
         """
 
         _id_list = []
         for page_number in range(self.last_page_number):
+            print ("Getting list of id's. {} % completed".format(
```

Comment:
Why do we need this print? Maybe it's better to use logging?

Reply:
Done